mantidimaging/gui/windows/main/view.py

```diff
@@ -99,6 +106,21 @@ def show_load_dialogue(self):
         self.load_dialogue = MWLoadDialog(self)
         self.load_dialogue.show()
 
+    def load_sample_log_dialog(self):
+        stack_selector = StackSelectorDialog(main_window=self,
+                                             title="Stack Selector",
+                                             message="Which stack is the log being loaded for?")
+        stack_selector.exec()
```

Comment:
This also needs to handle closing, e.g. `QDialog.Accepted != stack_selector.exec()`

Reply:
I really should have caught this! good spot!